middleware/common/source/communication/instance.cpp

```diff
@@ -241,20 +241,21 @@ namespace casual
 
                      if( common::environment::variable::exists( identity.environment))
                      {
-                        auto process = environment::variable::process::get( identity.environment);
-
-                        log::line( verbose::log, "process: ", process);
+                        if( auto process = environment::variable::get< process::Handle>( identity.environment))
+                        {
+                           log::line( verbose::log, "process: ", *process);
 
-                        if( ipc::exists( process.ipc))
-                           return process;
+                           if( ipc::exists( process->ipc))
+                              return *process;
+                        }
                      }
 
                      try
                      {
                         auto process = instance::fetch::handle( identity.id, directive);
 
-                        if( process && ! identity.environment.empty())
-                           environment::variable::process::set( identity.environment, process);
+                        //if( process && ! identity.environment.empty())
```

Comment:
Remove?

Reply:
I'll keep it rather. I think I commented that when I was testing to see what the consequences where if we didn't set these environment variables for children (in essence).